pages/dt.html

```diff
@@ -85,6 +98,7 @@
                         }
 
                         if ((!isOptOut) && (helpers.isEmpty(digiTrustUserJSON))) {
+                            log.debug('Create new DigiTrust Id on digitru.st domain');
                             // see if we can write and read cookies from within the iframe
                             DigiTrustCookie.createUserCookiesOnDigitrustDomain();
                             digiTrustUserJSON = DigiTrustCookie.getIdentityCookieJSON(cookieKey);

```

Comment:
suggest adding a debug statement here to the effect of `log.debug(digiTrustUserJSON);`. this seems like the most direct place to answer the question of "are cookies working at all".

Reply:
My only pause on this is exposing the User Id within debug logging. At a highly theoretical level the requestor needs a publisher key to expose Id. Do you have any thoughts on this?

Reply:
Addition of log statement that verifies valid ID without exposing actual ID.